controllers/storagecluster/cephtoolbox.go

```diff
@@ -117,21 +119,32 @@ func removeDuplicateTolerations(tolerations []corev1.Toleration) []corev1.Tolera
 	return list
 }
 
-func getMultusPublicNetwork(sc *ocsv1.StorageCluster) (string, error) {
+func (r *StorageClusterReconciler) getMultusPublicNetwork(sc *ocsv1.StorageCluster) (string, error) {
 	err := validateMultusSelectors(sc.Spec.Network.Selectors)
 	if err != nil {
 		return "", err
 	}
 
+	if sc.Spec.Network.Selectors["public"] == "" {
+		return "", nil
+	}
+
+	multusNetName := sc.Spec.Network.Selectors["public"]
+	nsInMultusName := strings.Split(multusNetName, "/")
+	if len(nsInMultusName) > 1 {
```

Comment:
Can you check that this is == 2
If it is > 2 return an error.

Reply:
Ack, updated the code